src/doc/python_scripting/quickrecipes.rst

```diff
@@ -1018,3 +882,136 @@ modify the legend.
     print(legend)
 
 
+Working with Color Tables
+-------------------------
+
+Sometimes it is helpful to create a new color table or manipulate an existing one.
+Color tables consist of ``ControlPoints`` which specify color and position in the color spectrum as well as a few other standard options.
+
+Existing color tables can retreived by name via ``GetColorTable`` as in: ::
+
+    hotCT = GetColorTable("hot")
+    print(hotCT)
+    # results of print
+    GetControlPoints(0).colors = (0, 0, 255, 255)
+    GetControlPoints(0).position = 0
+    GetControlPoints(1).colors = (0, 255, 255, 255)
+    GetControlPoints(1).position = 0.25
+    GetControlPoints(2).colors = (0, 255, 0, 255)
+    GetControlPoints(2).position = 0.5
+    GetControlPoints(3).colors = (255, 255, 0, 255)
+    GetControlPoints(3).position = 0.75
+    GetControlPoints(4).colors = (255, 0, 0, 255)
+    GetControlPoints(4).position = 1
+    smoothing = Linear  # NONE, Linear, CubicSpline
+    equalSpacingFlag = 0
+    discreteFlag = 0
+    categoryName = "Standard"
+
+The ``colors`` field of the ``ControlPoint`` is (R,G,B,A) in the range (0, 255).
```

Comment:
I dunno about you but I am not sure how much people will understand what `A` is here. It might make sense to simply spell that whole thing out including what *alpha* means (which is a measure of the color's transparency, correct?)

Reply:
Added information on RGBA meaning.